ga4gh/cli.py

```diff
@@ -384,6 +430,45 @@ def run(self):
             self._run(self._variantSetId)
 
 
+class SearchVariantAnnotationsRunner(
+        AnnotationFormatterMixin, AbstractSearchRunner):
+    """
+    Runner class for the variantannotationsg/search method.
+    """
+    def __init__(self, args):
+        super(SearchVariantAnnotationsRunner, self).__init__(args)
+        self._referenceName = args.referenceName
+        self._referenceId = args.referenceId
+        self._variantAnnotationSetId = args.variantAnnotationSetId
+        self._start = args.start
+        self._end = args.end
+
+        if args.featureIds == []:
+            self._featureIds = []
+        else:
+            self._featureIds = args.featureIds.split(",")
+
+        if args.effects == []:
+            self._effects = []
+        else:
+            self._effects = args.effects.split(",")
+
+    def _run(self, variantAnnotationSetId):
+        iterator = self._client.searchVariantAnnotations(
+            variantAnnotationSetId=variantAnnotationSetId,
+            referenceName=self._referenceName, referenceId=self._referenceId,
+            start=self._start, end=self._end,
+            featureIds=self._featureIds, effects=self._effects)
+        self._output(iterator)
+
+    def run(self):
+        if self._variantAnnotationSetId is None:
+            for annotationSet in self.getAllAnnotaionSets():
```

Comment:
I get an error here when I try to run this: have you defined getAllAnnotationSets?